probeinfo_api.yaml

```diff
@@ -524,7 +524,7 @@ components:
 
     V1Name:
       type: string
-      pattern: "^[a-z][a-z0-9-]{0,29}$"
+      pattern: "^[a-z][a-z0-9-]{0,39}$"
```

Comment:
@jklukas is this going to cause any problems downstream in the pipeline? Do you know where this "29 character limit" came from for this field?

Reply:
I'm not aware of any reason this would cause a problem.

It looks like this was originally introduced in https://github.com/mozilla/probe-scraper/commit/6399763b7309f11268cfe418cc1ae80ed53a8831 where the intent was simply to make sure the entries were kebab-case. I assume @mdboom chose the character limit based on the general size of values that existed at the time.

Reply:
Indeed: https://github.com/mozilla/probe-scraper/pull/113#issuecomment-527902928